src/cbapi/psc/threathunter/query.py

```diff
@@ -337,6 +337,23 @@ def __init__(self, doc_class, cb):
         self._query_token = None
         self._timeout = 0
         self._timed_out = False
+        self._sort_by = None
+        self._sort_direction = "ASC"
+
+    def sort_by(self, key, direction="ASC"):
```

Comment:
Note: We could make `key` a keyword parameter here, so that users can use the method to set only `direction` if so desired.

Reply:
I'd like to keep it required, so we're explicit about what key we're sorting by if we set the direction